### ISSUE TITLE
Add cloud sync

### ISSUE THREAD

**Issue by waozixyz** [naoxio/inbe#33]
would be great to be able to easily sync your progress online

**Comment by anhdres** [comment]
I agree, I don't think breathing sessions are such a sensitive information than dropbox or google drive sync would be an issue.
